daemon/lightning/lightning.go

```diff
@@ -27,4 +27,5 @@ type Client interface {
 	MonitorPaymentReception(ctx context.Context, rhash []byte) (Preimage, error)
 	GenerateInvoice(ctx context.Context, amountSats decimal.Decimal, expiry time.Duration, memo string) (paymentRequest string, rhash []byte, e error)
 	GenerateAddress(ctx context.Context) (string, error)
+	GetInvoicePreimage(ctx context.Context, rhash [32]byte) (Preimage, error)
```

Comment:
This function has the same signature as the `MonitorPaymentReception`, could that one be used instead of creating this new one?

Reply:
yes, you're right. Removed